sdcm/report_templates/results_base.html

```diff
@@ -196,7 +196,8 @@ <h4>
             {{ severity }} - [{{ events_summary.get(severity, 0) }}]
             </h4>
             {% for event in events %}
-                {{event}} <br/>
+<pre>{{event}}</pre>
```

Comment:
can you fix the indention of those two lines ?

Reply:
done